packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -1080,7 +1080,7 @@ function emitStdScalar(
     case "plainTime":
       return { type: "datetime", format: newScalar.format ?? "time" };
     case "offsetDateTime":
-      return { type: "datetime", format: newScalar.format ?? "rfc7231" };
+      return { type: "string" };
```

Comment:
Do you think we should apply this change to RLC also?

I have no strong preference but it seems reasonable to not generate Date for RLC if modular won't return date.

Core would use ISO format by default.
https://github.com/MaryGao/azure-sdk-for-js/blob/update-emitter-package-lock-file/sdk/core/core-client-rest/src/urlHelpers.ts#L50